### DISCUSSION TITLE
Tuta Feeds (like Wallabag)

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#7908] [Ideas]
I am searching for a service like Wallabag with rss feeds read in comfort way.

Possibility to save URL and/or the full article on database online (with offline capability).

Tags, etc.

**Comment by dermitdemdino** [comment]
I currently use [filen.io](https://github.com/FilenCloudDienste) in combination with [Joplin](https://github.com/laurent22/joplin). Activate in Filen WebDAV and synchronize Joplin with it. Works perfectly. I am happy with my combination of Tuta, Filen and Joplin.
